src/raven.erl

```diff
@@ -178,3 +186,39 @@ term_to_json_i(Term) when is_binary(Term); is_atom(Term) ->
     Term;
 term_to_json_i(Term) ->
     iolist_to_binary(io_lib:format("~120p", [Term])).
+
+request(Document, ?SENTRY_VER2) ->
+    Cfg = get_config(),
```

Comment:
you can provide Cfg from above and match on it to prevent calling it twice :)

Reply:
moved cfg to guard